gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.65.7
+version=29.65.8
```

Comment:
Please bump minor version if the change introduces new classes or new APIs. LinkedIn dependency check will only enforce bumping rest.li minor version not the patch version.

Reply:
Thanks for the catch. Updated this.